tests/errors/test_import_02.py

```diff
@@ -0,0 +1,3 @@
+from test_import_01 import X
+
+print(X)
```

Comment:
I think these new tests are not registered in `tests.toml`?

Reply:
Here, are the current outputs:
```console
$ 
warning: The module 'test_import/test_import_2' cannot be loaded
 --> tests/errors/test_import_02.py:1:1
  |
1 | from test_import.test_import_2 import X
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ imported here

semantic error: Type mismatch in annotation-assignment, the types must be compatible
 --> tests/errors/test_import/test_import_2.py:3:1
  |
3 | X: Const[i32] = 1.23
  | ^               ^^^^ type mismatch ('i32' and 'f64')
$ 
semantic error: Type mismatch in procedure call; the types must be compatible
 --> tests/errors/test_import_01.py:4:22
  |
4 | X: Const[f64] = test(5.0)
  |                      ^^^ type mismatch (passed argument type is f64 but required type is i32)

 --> tests/errors/test_import/test_import_1.py:3:13
  |
3 | def test(x: i32) -> i32:
  |             ^^^ type mismatch (passed argument type is f64 but required type is i32)
```